.github/global.yml

```
@@ -230,6 +232,8 @@
   description: related to producing reproducible results
   color: "86C579"
   aliases: [reproducibility]
+- name: tag::ux
+  description: related to user experience
```

Tags also use the same color:

```suggestion
  description: related to user experience
  color: "86C579"
```